app/javascript/packs/sidebar.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener('turbolinks:load', function(){
+  var sidebar = document.getElementById("sidebar");
+  var close = document.querySelector(".sidebar-close");
+  var menu = document.getElementById("menu");
+  
+  function openNav() {
+    sidebar.classList.remove('hidden');
+  }
+  
+  function closeNav() {
+    sidebar.classList.add('hidden');
+  }
+
+  function isClose(){
+    sidebar.classList.contains('hidden');
+  }
+  
+  menu.addEventListener('click',openNav);
+
+  close.addEventListener('click',closeNav)
+
+  document.addEventListener('keydown', e => {
+    if (e.keyCode == 27 && !this.isClose)
```

Comment:
It's not clear what the 27 means here.

A good practice is naming this kind of "magic numbers" by moving them to a vars that give us an idea of why are they used.

Here is a post that explains this better 😄  https://refactoring.guru/replace-magic-number-with-symbolic-constant